extensions/amp-story/0.1/amp-story.js

```diff
@@ -417,27 +417,36 @@ export class AmpStory extends AMP.BaseElement {
       this.performTapNavigation_(direction);
     });
 
-    const gestures = Gestures.get(this.element,
-        /* shouldNotPreventDefault */ true);
+    this.win.document.addEventListener('keydown', e => {
+      this.onKeyDown_(e);
+    }, true);
 
-    gestures.onGesture(SwipeXYRecognizer, e => {
+    this.getViewport().onResize(debounce(this.win, () => this.onResize(), 300));
```

Comment:
Did some unrelated housekeeping here. `this.boundOnResize_` was assigned but not used elsewhere.